packages/docs/docs/careplans/loinc.md

```diff
@@ -0,0 +1,373 @@
+---
+keywords:
+  - loinc
+  - diagnostics
+  - lab
+  - lab codes
+  - diagnostic codes
+  - LIS/LIMS
+tags:
+  - lab
+  - diagnostics
+  - loinc
+  - care plans
+  - LIS
+  - LIMS
+---
+
+# LOINC Codes
+
+LOINC (Logical Observation Identifiers Names and Codes) was established with the intention of defining a universal standard for identifying clinical data in electronic reports. The overall scope of LOINC is anything you can **test**, **measure**, or **observe** about a patient.
```

Comment:
Minor: Consider making this first reference to the word "LOINC" into a link to https://loinc.org